source/extensions/filters/http/cache/cache_filter_logging_info.cc

```diff
@@ -0,0 +1,79 @@
+#include "source/extensions/filters/http/cache/cache_filter_logging_info.h"
+
+#include "absl/strings/str_format.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Cache {
+
+absl::string_view cacheLookupStatusToString(CacheLookupStatus status) {
+  switch (status) {
+  case CacheLookupStatus::Unknown:
+    return "Unknown";
+  case CacheLookupStatus::CacheHit:
+    return "CacheHit";
+  case CacheLookupStatus::CacheMiss:
+    return "CacheMiss";
+  case CacheLookupStatus::StaleHitWithSuccessfulValidation:
+    return "StaleHitWithSuccessfulValidation";
+  case CacheLookupStatus::StaleHitWithFailedValidation:
+    return "StaleHitWithFailedValidation";
+  case CacheLookupStatus::NotModifiedHit:
+    return "NotModifiedHit";
+  case CacheLookupStatus::RequestNotCacheable:
+    return "RequestNotCacheable";
+  case CacheLookupStatus::RequestIncomplete:
+    return "RequestIncomplete";
+  case CacheLookupStatus::LookupError:
+    return "LookupError";
+  default:
```

Comment:
nit: drop the default and put the return after the `}` so the compiler will warn you if you are missing cases.